Source/monstdat.h

```diff
@@ -75,42 +77,42 @@ typedef enum monster_resistance {
 } monster_resistance;
 
 typedef struct MonsterData {
-	Sint32 width;
-	Sint32 mImage;
+	const char *mName;
 	const char *GraphicType;
-	bool has_special;
 	const char *sndfile;
+	const char *TransFile;
+	Uint16 width;

```

Comment:
I think 160 is about as big as they get, so potentially there is a bit more that could be cut away, but not very future proof of cause.